cmd/jujud/agent/model/manifolds.go

```diff
@@ -174,11 +174,11 @@ func commonManifolds(config ManifoldsConfig) dependency.Manifolds {
 		}),
 		// Cloud credential validator runs on all models, and
 		// determines if model's cloud credential is valid.
-		credentialValidatorFlagName: ifNotUpgrading(ifNotDead(credentialvalidator.Manifold(credentialvalidator.ManifoldConfig{
+		credentialValidatorFlagName: credentialvalidator.Manifold(credentialvalidator.ManifoldConfig{
```

Comment:
So we can lose the ifNotDead?
Does it make sense to run this worker on a dead model?

Reply:
I do not think that there is a need for us to prevent this worker's run on a Dead model... What if we do need to do something that would require a valid cloud credential?
At this stage, I do not think it hurts us.